app/controllers/restaurants_controller.rb

```diff
@@ -11,4 +11,24 @@ def index
   def show
     @restaurant = Restaurant.find(params[:id])
   end
+
+  def new
+  end
```

Comment:
More 'railsy' way of doing this is
```ruby
def new
    @restaurant = Restaurant.new
end
```
This will allow you to simplify the `form_for` method in your view to: 
```haml
= form_for @restaurant  do |f|
```